package.json

```
@@ -4,7 +4,10 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "install:all": "npm i && npm i --prefix src/functions",
```

@chiharakenta @tk-ozawa @Kazu950 
subdirectory でも `npm i` できるようにしました
今後 `gas/*` で同じようなことをしたい場合は以下のようにしてください
```suggestion
    "install:all": "npm i && npm i --prefix src/functions && npm i --prefix src/gas",
```